checks.d/mcache.py

```diff
@@ -162,18 +163,18 @@ class Memcache(AgentCheck):
     SERVICE_CHECK = 'memcache.can_connect'
 
     def get_library_versions(self):
-        return {"memcache": memcache.__version__}
+        return {
+            "memcache": pkg_resources.get_distribution("python-binary-memcached").version
+        }
 
     def _get_metrics(self, client, tags, service_check_tags=None):
         try:
-            raw_stats = client.get_stats()
+            stats_by_host = client.stats()
```

Comment:
@truthbk: The reason I have the 1-length assertion is that the Memcached check is configured to only talk to one server (per [`mcache.yaml`](https://github.com/DataDog/dd-agent/blob/74f674645b64ac47f51d94cd847a0d562414dbf0/conf.d/mcache.yaml.example)), so getting a 0-sized dict would indicate the server didn't report stats and a greater-than-1-sized dict would indicate multiple servers reporting stats/the single server misreporting stats. Please let me know if you think a different approach should be taken! 🙃